test/serialization/JSONACSets.jl

```diff
@@ -42,7 +42,8 @@ add_parts!(g, :V, 3)
 add_parts!(g, :E, 2, src=[1,2], tgt=[2,3], weight=[0.5,1.5])
 @test roundtrip_json_acset(g) == g
 
-SchLabeledDDS = BasicSchema([:X], [(:Φ,:X,:X)], [:Label], [(:label,:X,:Label)])
+SchLabeledDDS = BasicSchema([:X], [(:Φ,:X,:X)], [:Label], [(:label,:X,:Label)], 
+                            [(:cycle4,:X,:X,((),(:Φ,:Φ,:Φ,:Φ)))])
```

Comment:
Can we either make a new test or rename the schema, since with the new equation it's no longer than the schema for discrete dynamical systems?